README.md

```diff
@@ -1,25 +1,30 @@
 
-To estimate the minimum length scale of void regions in a 2d binary image, the binary image is inverted after binarization, i.e., $\rho \rightarrow \neg \rho$ so that solid and void regions are interchanged. Subsequent procedures are the same as described above. This approach is equivalent to computing $\mathcal{C}_d(\rho) \oplus \rho$ in Step 2 and test the overlap with interior pixels of void regions in $\rho$ in Step 3. Here $\mathcal{C}_d(\rho)$ denotes morphological closing.
+For a 1d binary image, the algorithm simply finds the minimum length among all solid or void segments.
 
-The minimum length scale of $\rho$ is the minimum between the minimum length scales of solid and void regions. To obtain this minimum, one can compute the minimum length scales of solid and void regions separately and take the minimum. A more efficient approach is to compute $\mathcal{O}_d(\rho) \oplus \mathcal{C}_d(\rho)$ in Step 2 and test the overlap with the union of interior pixels of solid and void regions in $\rho$ in Step 3.
+## Note on Accuracy
 
-The accuracy of the minimum lengthscale is limited by the finite resolution of the image: it is obvious that lengthscales below the single-pixel level cannot be measured, but even few-pixel scales cannot always be distinguished from "staircased" discretizations of curved surfaces.  For example, a 90° sharp corner corresponds to a lengthscale (radius of curvature) of *zero* at infinite resolution, but at finite resolution it cannot be distinguished from a discretization of a rounded corner with a radius of a few pixels, so `imageruler` will return a lengthscale proportional to (but greater than) the pixel size in that case.  In general, the measured lengthscale should be viewed as having an "error bar" on the few-pixel level.
+The accuracy of the minimum length scale computed by Imageruler is limited by the finite resolution of the input image. A fundamental feature is that length scales smaller than a single pixel cannot be measured. Also, in certain situations, length scales of a few pixels may be indistinguishable from discretization artifacts such as the "staircasing" of curved surfaces. As an example, a sharp 90° corner corresponds to a length scale (radius of curvature) of *zero* at infinite resolution. However, at a finite resolution, a sharp corner will tend to be rounded with a radius of curvature of a few pixels. Imageruler will therefore return a length scale proportional to the pixel size. In general, the measured length scale should be viewed as having an "error bar" on the few-pixel level.
```

Comment:
```suggestion
The accuracy of the minimum length scale computed by Imageruler is limited by the finite resolution of the input image. A fundamental feature is that length scales smaller than a single pixel cannot be measured. Also, in certain situations, length scales of a few pixels may be indistinguishable from discretization artifacts such as the "staircasing" of curved surfaces. As an example, a sharp 90° corner corresponds to a length scale (radius of curvature) of *zero* at infinite resolution. However, at a finite resolution, a sharp corner is indistinguishable from one that is rounded with a radius of curvature of a few pixels. Imageruler will therefore return a length scale proportional to the pixel size rather than zero. In general, the measured length scale should be viewed as having an "error bar" on the few-pixel level.
```